docs/guides/qiskit-code-assistant-local.mdx

```diff
@@ -24,18 +24,80 @@ The Open Source Qiskit Code Assistant models are available in <DefinitionTooltip
+4. [qiskit/qwen2.5-coder-14b-qiskit](https://huggingface.co/Qiskit/qwen2.5-coder-14b-qiskit) - Released June 2025
+
+#### GGUF Models (Recommended for Personal Environments/Laptops)
+
+GGUF format models are optimized for local use and require less computational resources:
+
+1. [qiskit/granite-3.3-8b-qiskit-GGUF](https://huggingface.co/Qiskit/granite-3.3-8b-qiskit-GGUF) - Released June 2025
+2. [qiskit/granite-3.2-8b-qiskit-GGUF](https://huggingface.co/Qiskit/granite-3.2-8b-qiskit-GGUF) - Released June 2025
+3. [qiskit/qwen2.5-coder-14b-qiskit-GGUF](https://huggingface.co/Qiskit/qwen2.5-coder-14b-qiskit-GGUF) - Released June 2025
+
+#### Qiskit Versions Used for Training
+
+<table
+  style={{
+    display: 'inline-table',
+    borderCollapse: 'separate',
+    borderSpacing: 0,
+    fontFamily: "Inter, -apple-system, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif",
+    boxShadow: '0 6px 18px rgba(12, 20, 29, 0.06)',
+    borderRadius: '12px',
+    overflow: 'hidden',
+    tableLayout: 'auto',
+    boxSizing: 'border-box',
+    margin: '16px 0'
+  }}
+>
+  <thead>
+    <tr>
+      <th style={{textAlign: 'left', padding: '10px 14px', background: '#e8f0f8', color: '#475569', fontWeight: 600, fontSize: '13px', borderBottom: '1px solid rgba(15,23,42,0.08)'}}>
+        Model
+      </th>
+      <th style={{textAlign: 'center', padding: '10px 14px', background: '#e8f0f8', color: '#475569', fontWeight: 600, fontSize: '13px', borderBottom: '1px solid rgba(15,23,42,0.08)'}}>
+        Trained on Qiskit Version
+      </th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr style={{background: '#ffffff'}}>
+      <td style={{padding: '10px 14px', fontWeight: 700, color: '#0f172a', fontSize: '14px'}}>mistral-small-3.2-24b-qiskit</td>
+      <td style={{padding: '10px 14px', fontWeight: 700, textAlign: 'center', fontFamily: 'ui-monospace, SFMono-Regular, Menlo, Monaco, monospace', fontSize: '14px'}}>2.1</td>
+    </tr>
+    <tr style={{background: '#fbfdff'}}>
+      <td style={{padding: '10px 14px', color: '#0f172a', fontSize: '14px'}}>granite-3.3-8b-qiskit</td>
+      <td style={{padding: '10px 14px', textAlign: 'center', fontFamily: 'ui-monospace, SFMono-Regular, Menlo, Monaco, monospace', fontSize: '14px'}}>2.0</td>
+    </tr>
+    <tr style={{background: '#ffffff'}}>
+      <td style={{padding: '10px 14px', color: '#0f172a', fontSize: '14px'}}>granite-3.2-8b-qiskit</td>
+      <td style={{padding: '10px 14px', textAlign: 'center', fontFamily: 'ui-monospace, SFMono-Regular, Menlo, Monaco, monospace', fontSize: '14px'}}>2.0</td>
+    </tr>
+    <tr style={{background: '#fbfdff'}}>
+      <td style={{padding: '10px 14px', color: '#07102a', fontSize: '14px'}}>qwen2.5-coder-14b-qiskit</td>
+      <td style={{padding: '10px 14px', textAlign: 'center', fontFamily: 'ui-monospace, SFMono-Regular, Menlo, Monaco, monospace', fontSize: '14px'}}>2.0</td>
+    </tr>
```

Comment:
It would be better to sort the table by actual benchmark metrics, not alphabetically by the name of the model. With that visual information it can be easier for the users to pick which model they can use.

Even more, we may want to consider consolidating the information about the release date and Qiskit version used during training in the same table